core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/GlobalReplayManager.scala

```diff
@@ -0,0 +1,23 @@
+package edu.uci.ics.amber.engine.architecture.controller
+
+import edu.uci.ics.amber.engine.common.virtualidentity.ActorVirtualIdentity
+
+import scala.collection.mutable
+
+class GlobalReplayManager(onRecoveryStart: () => Unit, onRecoveryEnd: () => Unit) {
```

Comment:
we use `start` and `complete` for all other lifecycles. can we align the naming?

Reply:
renamed.